src/queue/check.ts

```diff
@@ -37,12 +38,13 @@ export async function checkVideoFolders() {
         } else {
           logger.log(`Found matching file ${path}`);
           const existingScene = await Scene.getSceneByPath(path);
+          if (config.DELETE_UNVERIFIED && existingScene) Scene.flagExisting(existingScene, true)
```

Comment:
missing await, unless you think we can intentionally run this in the background ?

```suggestion
          if (config.DELETE_UNVERIFIED && existingScene) await Scene.flagExisting(existingScene, true)
```